dist/Safe/t/safe3.t

```diff
@@ -1,23 +1,19 @@
 #!perl -w
 
-BEGIN {
-    require Config; Config->import;
-    if ($Config{'extensions'} !~ /\bOpcode\b/
-	&& $Config{'extensions'} !~ /\bPOSIX\b/
-	&& $Config{'osname'} ne 'VMS')
-    {
-	print "1..0\n";
-	exit 0;
-    }
-}
+use Config;
+use Test::More
+    $Config{'extensions'} =~ /\bOpcode\b/
+	|| $Config{'extensions'} =~ /\bPOSIX\b/
+	|| $Config{'osname'} eq 'VMS'
+    ? (tests => 2)
+    : (skip_all => "no Opcode and POSIX extensions and we're not on VMS");
```

Comment:
While I don't think you've changed the meaning of the logic here, I think your change makes a mistake in the original code more obvious.

The test code requires both `POSIX` and `Opcode`, but the condition allows the tests to run with either available, not requiring both.

Rather than doing this complex `use`, perhaps:
```
use Test::More;
BEGIN {
    plan skip_all => "no Opcode and POSIX extensions and we're not on VMS"
       unless $Config{'extensions'} =~ /\bOpcode\b/
	    && $Config{'extensions'} =~ /\bPOSIX\b/
	    || $Config{'osname'} eq 'VMS';
}
plan tests => 2;
```
though that's a matter of taste.

I haven't been able to figure out what the osname==VMS test is for.